assisted-events-scrape/tests/integration/test_integration.py

```diff
@@ -28,7 +35,5 @@ def _wait_for_elastic(self):
     def _is_elastic_ready(self) -> bool:
         is_elastic_ready = self._es_client.indices.exists(index=self._config.index)
         if is_elastic_ready:
```

Comment:
We remove the sleep call now, as document are synced by a refresh call, however this can still technically be flaky (hopefully low probability), as it is not necessarily true that when the index is created all documents have already been inserted, especially with current code which inserts one document at the time.

Reply:
I guess only way to make it stable is adding a loop with ``waiting.wait`` that expects a certain amount of docs, right?

Reply:
Yes, maybe I could do it something like this:
* init waits that elastic has the index
* assertion waits (less) that has the right amount of docs
WDYT @osherdp ?

Reply:
That sounds about right

Reply:
Either way not a must for this PR, so whatever works for you

Reply:
Already implemented :)